arches/app/templates/views/components/widgets/datepicker.htm

```diff
@@ -87,16 +89,19 @@
 </div>
 <div>
     <div class="form-group col-xs-12 pad-no crud-widget-container">
-        <div style="pad-hor">
+        <div class="pad-hor">
             <div class="input-group date">
-                <input type="text" data-bind="value: defaultValue, placeholder: placeholder, datepicker: {format: dateFormat, viewMode: viewMode, maxDate: maxDate, minDate: minDate}" class="form-control input-md" style="height: 32px;"><span class="input-group-addon"><i class="fa fa-calendar fa-lg"></i></span>
+                <input type="text" data-bind="value: defaultValue, placeholder: placeholder, attr: {'aria-label': placeholder},
+                    datepicker: {format: dateFormat, viewMode: viewMode, maxDate: maxDate, minDate: minDate}" class="form-control input-md" style="height: 32px;"
+                    ><span class="input-group-addon date-icon"><i class="fa fa-calendar fa-lg date-icon"></i></span>
             </div>
         </div>
     </div>
     <div class="toggle-container">
-        <span class="switch switch-small" data-bind="css: {'on': getdefault}, click: setdefault"><small></small></span>
+        <span id="default-date-switcher" class="switch switch-small" data-bind="css: {'on': getdefault}, onEnterkeyClick, onSpaceClick, click: setdefault, 
```

Comment:
we should add the `aria-describedby` property here that references the subtitle below